targets/frontend/src/components/contributions/questionList/QuestionRow.tsx

```diff
@@ -26,7 +20,7 @@ export const QuestionRow = (props: { row: QueryQuestion }) => {
       hover
     >
       <TableCell component="th" scope="row">
-        {row.content}
+        {row.order} - {row.content}
```

Comment:
un petit strong sur le number pour le rendre plus lisible ?
```suggestion
        <strong>{row.order}</strong> - {row.content}
```